config/locales/en/brexit_checker/results.yml

```diff
@@ -1,13 +1,12 @@
 en:
   brexit_checker:
     results:
-      title: "Get ready for a no-deal Brexit: Your results"
```

Comment:
This confused me - I think we remove the title in this commit and it back somewhere else?